### PR TITLE
Use skill name instead of skill path in build_test_configuration

### PR DESCRIPTION
**Pull request by forslund**
This finds the skill name from the skill path and uses that when creating the test_skill.yml which should be preferable as noted by @krisgesling in #1530

### CODE REVIEW

**Comment by devops-mycroft** [comment]
Voight Kampff Integration Test Succeeded  ([Results](https://reports.mycroft.ai/skills/PR-1550))

File: build_test_config.py [hunk 1]
```diff
@@ -57,6 +58,22 @@ def get_pull_request_submodule(pull_request_diff):
...
 
+def get_skill_submodule_name(skill_submodule_path):
+    """Find the skill name from a skill submodule path."""
+    with open('.gitmodules') as f:
+        for line in f:
+            name_match = re.match(r"\[submodule \"(?P<entryname>.+)\"\]", line)
+            if name_match:
+                name = name_match.groups()[0]
+            if line.strip() == f'path = {skill_submodule_path}':
+                skill_url = f.readline().split(' = ')[1]
```
Comment (krisgesling) [inline build_test_config.py]: Does this get used at all?

**Review by krisgesling — APPROVED** [review]
Thanks for this - and good idea to fallback to the submodule path still.

Potentially an unnecessary line left in, but it certainly doesn't hurt anything.

File: build_test_config.py (hunk 1 above)
Comment (forslund) [reply build_test_config.py]: Typical copy-pasta. Will remove

**Comment by devops-mycroft** [comment]
Voight Kampff Integration Test Succeeded  ([Results](https://reports.mycroft.ai/skills/PR-1550))

**Pull request merged**